content/learn/book/welcome/_index.md

```diff
@@ -6,4 +6,33 @@ template = "book-section.html"
 page_template = "book-section.html"
 +++
 
-TODO: Briefly discuss what Bevy is.
+Hi! Thanks for choosing the Bevy Game Engine! Our community is glad to have you!
+
+Within a few minutes, you will be making some games and learning more of what this amazing engine has to offer, but before we dive into that, we need an introduction.
+
+## What Bevy is (and what it is not)
+
+The Bevy Game Engine is a refreshingly simple data-driven game engine built in Rust that is completely [open-source](https://github.com/bevyengine/bevy). It is capable of 2D and 3D games, uses an Entity Component System and allows for easy enhancement and customization.
+
+It is constantly being updated and worked on by its creator, [cart](https://github.com/cart) and the engine's increasing commmunity. **Due to this the engine is not currenty production-ready and should NOT be used for time-sensitive projects**
+
+In quicker terms, Bevy is:
+- A easy-to-use game-making tool
+- Capable of 2D and 3D
+- Data-Driven
+- Easily extendable
+
+And isn't:
+- A fully-featured product
+- Intended for use with big teams of people with jobs on the line
+- Stable
+
+## General Recommendations
+
+While this book will provide context for some Rust capabilities, it does NOT include a lot of the best practices that the language and this may inhibit your learning. 
+
+We highly recommend reading some form of Rust documentation. The best resource to choose is probably [The Rust Book](https://docs.rust-lang.org/book), as it provides projects and examples.
+
+However, if you like to get your hands dirty quick, you can try [Rust by Example](https://docs.rust-lang.org/stable/rust-by-example) or [Rustlings](https://github.com/rust-lang/rustlings/).
```

Comment:
I like these additional resources a lot; I think we should aim to add these to #176.